proto/types.proto

```diff
@@ -11,3 +11,15 @@ message ExtraData {
 	bytes JsonSpec = 1;
 	google.protobuf.Any RuncOptions = 2;
 }
+
+// Message to specify network config for a Firecracker VM
+message FirecrackerNetworkConfig {
```

Comment:
I think I'd prefer that this was `FirecrackerNetworkInterface` to match the `firecracker.NetworkInterfaces` type.

Reply:
good suggestion. +1 for having same names everywhere.

Reply:
Should be fixed now, can you take another look?